app/config/dev-config.yaml

```diff
@@ -0,0 +1,34 @@
+# dev config
```

Comment:
what's the difference between this and `dev-config-pub.yaml`?

Reply:
one is exposing everything only to localhost, the second one is using also appengine:
https://github.com/dart-lang/pub-dev/pull/4588/files#diff-b9dd2d6cb9598fc3f73907fc582d6d8e9f9efdf89438e2bbf646b2ab658031c5R20
lint 20 in example